README.md

```diff
@@ -28,19 +25,18 @@
 
 <a href="https://xn--931a.moe/"><img src="https://github.com/misskey-dev/misskey/blob/develop/assets/ai.png?raw=true" align="right" height="320px"/></a>
 
-## ✨ Features
+## ✨ Highlights
 - **ActivityPub support**\
-	It is possible to interact with other software.
+	Federates well with Mastodon, Pleroma and more.
```

Comment:
Interoperable with other Fediverse software (including Mastodon, Pixelfed, Pleroma, and more).

Reply:
I find the word "Interoperable" much less precise than "Federate", but don't really mind